server/lobby.js

```diff
@@ -193,25 +195,30 @@ class Lobby {
         let filteredUsers = userList;
 
         if(socket.user) {
-            filteredUsers = _.reject(userList, user => {
-                return _.contains(socket.user.blockList, user.name.toLowerCase());
+            filteredUsers = userList.filter(user => {
+                return !socket.user.blockList.includes(user.name.toLowerCase());
             });
         }
 
         socket.send('users', filteredUsers);
     }
 
     broadcastGameList(socket) {
-        let sockets = socket ? [socket] : this.sockets;
-        _.each(sockets, socket => {
+        let sockets = {};
```

Comment:
Previous logic was to broadcast to all connected sockets if a single socket wasn't passed. This now broadcasts to no sockets

Reply:
This should be functionally equivalent, it just works around the old issue where it was either an array or an object, and underscore Just Dealt With It(tm)